include/global.h

```diff
@@ -15,6 +15,7 @@
 #include "constants/pokemon.h"
 #include "constants/easy_chat.h"
 #include "constants/expansion_branches.h"
+#include "constants/trainer_hill.h"
```

Comment:
> I would put `#include "constants/expansion_branches.h"` below everything

Why? 🤔 

It makes more sense to put `trainer_hill.h` imo, since it was added just recently.

Reply:
Fair enough.

Reply:
Remember to add the use of `u32 trainerHillTimes[NUM_TRAINER_HILL_MODES];`, as that's why the trainer hill header file was added